src/oscar/core/customisation.py

```diff
@@ -73,11 +73,14 @@ def fork_app(label, local_folder_path, local_app_subpackage=None, logger=None):
         if not isinstance(app_config, OscarConfig):
             raise ValueError("There is no Oscar app with the label '{}'".format(label))
 
-    # Check local_folder_path is current folder
+    # Remove trailing slash from folder path
+    local_folder_path = local_folder_path.rstrip('/')
+
+    # Check if local_folder_path is current folder
     if local_folder_path == '.':
         local_folder_path = ''
 
-    local_apps_package = local_folder_path.replace('/', '.')
+    local_apps_package = local_folder_path.lstrip('/').replace('/', '.')
```

Comment:
Why do we need to remove leading slash here?

Reply:
Has been able to re-create the case this line aims to fix, which proves necessity of test coverage.